packages/forms-web-app/src/controllers/your-planning-appeal.js

```diff
@@ -0,0 +1,21 @@
+const logger = require('../lib/logger');
+const { VIEW } = require('../lib/views');
+const appealModel = require('../models/appealModel');
+
+exports.getYourPlanningAppeal = async (req, res, next) => {
+  const { appealId } = req.params;
+  let appeal;
+
+  try {
+    appeal = await appealModel.getAppeal(appealId);
+  } catch (error) {
+    if (error.response.status === 404) return next();
+
+    logger.error(error);
+    return next(error);
+  }
+
+  res.render(VIEW.YOUR_PLANNING_APPEAL, {
+    appeal,
+  });
+};
```

Comment:
Please could the path to this file be:

`packages/forms-web-app/src/controllers/your-planning-appeal/index.js`

This will help with https://pins-ds.atlassian.net/browse/AS-1607 where there needs to be sub pages under this route. 

Thanks